linux-tkg-config/prepare

```diff
@@ -179,6 +179,14 @@ _set_kver_internal_vars() {
   # examples: "600", "515", "504" we use this variable to have proper comparisons
   _kver="${_kver_split[0]}${_kver_split[1]}"
 
+  if [ $_kver -le 605 ]; then
+    # chosen kernel <= 6.5
+    _default_cpu_sched="cfs"
+  else
+    # chosen kernel >= 6.6
+    _default_cpu_sched="eevdf"
+  fi
+
   echo -e "_basekernel='$_basekernel'\n_basever='$_basever'\n_sub='$_sub'\n_kver='$_kver'" > "$_where"/BIG_UGLY_FROGMINER
```

Comment:
I think we need to also save it in the file `BIG_UGLY_FROGMINER` too, so it's not lost when contexts change
```suggestion
# add it the the file BIG_UGLY_FROGMINER (because the bash script is sometimes called from different contexts where the global variables get reset, so this file is sourced every time)
 echo -e "_basekernel='$_basekernel'\n_basever='$_basever'\n_sub='$_sub'\n_kver='$_kver'\n_default_cpu_sched='$_default_cpu_sched'" > "$_where"/BIG_UGLY_FROGMINER
```